app/views/users/_form.html.erb

```diff
@@ -38,7 +38,7 @@
     <%= form.label :user_class_year, "Class Year", style: "display: block" %>
     <!-- <%= form.text_field :user_class_year %> -->
     <% current_year = Date.today.year %>
-    <%= form.select :user_class_year, options_for_select((current_year - 100..current_year + 4).map { |year| [year.to_s, year] }, @user.user_class_year) %>
+    <%= form.select :user_class_year, options_for_select((current_year - 100..current_year + 4).to_a.reverse.map { |year| [year.to_s, year] }, @user.user_class_year), include_blank: true %>
```

Comment:
I would consider moving the date calculations to the Controller to make this view less cluttered.